packages/flutter/lib/src/animation/tween.dart

```diff
@@ -37,42 +37,42 @@ abstract class Animatable<T> {
 }
 
 class _AnimatedEvaluation<T> extends Animation<T> with AnimationWithParentMixin<double> {
-  _AnimatedEvaluation(this.parent, this._evaluatable);
+  _AnimatedEvaluation(this.parent, this._evaluable);
```

Comment:
Based on this graph, I think it's ok 
https://english.stackexchange.com/a/312764